src/app/shapes/Ellipse.ts

```diff
@@ -0,0 +1,35 @@
+import * as Konva from "konva";
+import {Shape} from "./Shape";
+
+export class Ellipse extends Shape{
+
+  public instance: Konva.Ellipse;
```

Comment:
`instance` is a variable in the Shape superclass, so you shouldn't need to define it here.

Reply:
We need to define it so that we can use Ellipse specific functionality.